app/views/patients/_form.html.slim

```diff
@@ -1,152 +1,151 @@
-        .medium-4.columns
+        .medium-6.columns
+          = f.input :is_on_active_duty, label: "Currently on Active Duty"
+          = f.input :sci_service_connected, label: "SCI is service connected"
+          = f.input :my_healthevet_messaging
+        .medium-6.columns
+          label.control-label.radio Theater of service
+          = f.input_field :theater_of_service_id,
+          collection: Domain::TheaterOfService.cached_all,
+          as: :radio_buttons,
+          item_wrapper_class: 'radio-inline'
+      .row
+        hr
+        .medium-6.columns
           = f.association :assigned_vamc, :label => "Assigned VAMC",
           collection: Domain::VaMedicalCenter.cached_all,
           prompt: "Select one"
-        .medium-4.columns
-          / TOODO: Should be conditional on last radio selected
-          = f.input :benefits_waiver_exemption_date, as: :date_picker, placeholder: "will be conditional upon travel status"
-      .row
         .medium-12.columns
           label.control-label.radio Travel status
           = f.input_field :travel_status_id,
             collection: Domain::TravelStatus.cached_all,
             as: :radio_buttons,
             item_wrapper_class: 'radio-inline'
-    .callout
-      h4 SCI Hub and Physician
-      h4 Assigned 
-      .row
-        .medium-6.columns
-          = f.association :assigned_sci_hub,
-          collection: Domain::SciHub.cached_all,
-          label: "Location"
-        .medium-3.columns
-          = f.input :assigned_sci_hub_physician_first_name, label: "Physician first name"
-        .medium-3.columns
-          = f.input :assigned_sci_hub_physician_last_name, label: "Physician last name"
-      h4 Preferred (if different)
-      .row
-        .medium-6.columns
-          = f.association :preferred_sci_hub,
-          collection: Domain::SciHub.cached_all,
-          label: "Location"
-        .medium-3.columns
-          = f.input :preferred_sci_hub_physician_first_name, label: "Physician first name"
-        .medium-3.columns
-          = f.input :preferred_sci_hub_physician_last_name, label: "Physician last name"
+        .medium-6.left.columns#benefitsOther.hide
+          /Will show conditionally on last radio selected
+          = f.input :benefits_waiver_exemption_date, 
+            as: :date_picker
     .callout
       h4 Primary Care
       .row
         .medium-12.columns
-          / This could be in a later flow displaying the information (w/ address and physician)
           label.control-label.radio.radio-inline Primary Care Physician
           = f.input_field :principle_pcp_va_nonva_id,
           label: "Primary Care Physician",
           collection: Domain::PrinciplePcpType.cached_all,
           as: :radio_buttons,
           item_wrapper_class: 'radio-inline'
-      h4 VA Facility
-      .row
-        .medium-6.columns
-          / TODO: Should this be actually facility?
-          = f.association :va_facility, label: "Facility"
```

Comment:
This is missing a comma after "Facility".  Not sure how it's working, but we should have it...